src/main/java/com/artipie/rpm/Repomd.java

```diff
@@ -139,6 +139,12 @@ private Completable loadRepomd(final Path file) {
      * @param act Action
      * @param repomd The temp
      * @return Completion or error signal
+     * @todo #41:30min Continue refactoring Repomd.performUpdate method
```

Comment:
@victornoel You could improve the architecture right now instead just suggesting it, for example, instead just putting this puzzle saying "we might move some parts of this into other classes" you could put puzzles in the parts of the code that you think that could be in another classes and ask the user to implement this new design